### PR TITLE
feat: add npm start to package.json

### PR DESCRIPTION
**Pull request by Jemair**
add `npm start` for original react developers
add `.idea/` in `.gitignore` for webstorm users

### CODE REVIEW

**Comment by Jemair** [comment]
I added `npm start` for these reasons:

1. The most generally used react boilerplate, c-r-a uses `npm start` to start a dev-server. For which, in my opinion, there are a great mount of *react developer* being used to use this command. Especially those who had never approached `vue-cli`, by whom, I meant *original react developers*

2. The fact that the npm team adds `npm start` as an alias for `npm run start` shows the specialty of this command. I think it is better recommended by the offical.

3. It is only a tiny modify to make more people happy, why not? During my using this boilerplate, timer and time again I input `npm start` to start it which may be kind of 'muscle memory'? 🌸🐣


The second commit came from my learning of c-r-a, trying to figure out how could they achieve to reuse the tab(As you can see I just copied some code).  Which I find it very useful as the daily I use this boilerplate is like this:
image

so maybe we can make some people rich firstly? there are so many coders using chrome + mac after all, meanwhile we have the B plan using the `opn` plugin. and of course i don't think the current way is the best way to organize the codes.

To sum up, I think the 1st commit is a micro one but can improve a lot of developing experience. And the 2nd one is a dependance on you, neither choice you made I will totally understand

**Comment by SidKwok** [comment]
Um.. I was convinced lol. 

We can take it as our temporary solution for reusing tab. Will improve it since we had a better option.

Thanks again.
